dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,7 @@ found in the LICENSE file. -->
         android:label="macrobenchmarks"
         android:icon="@mipmap/ic_launcher">
         <activity
+            android:exported="true"
```

Comment:
I see this line popping up when developers create a new benchmark. Is it needed? Is it an issue? Should I just let it go through?

Reply:
This is a config change needed to run on the newest version of android. Adding an automatic migration is tracked in https://github.com/flutter/flutter/issues/102183